### DISCUSSION TITLE
Does scikit-build support ABI3

### DISCUSSION THREAD

**Discussion by Erotemic** [scikit-build/scikit-build#853] [Q&A] [answered]
Is there a recommended way to use scikit-build to compile ABI3 wheels? Ideally in a way that is compatible with [cibuildwheel](https://cibuildwheel.readthedocs.io/en/stable/faq/#abi3)?

**Comment by henryiii** [comment, marked as answer]
Scikit-build-core will support this in version 0.2 - PR at https://github.com/scikit-build/scikit-build-core/pull/102. CMake 3.26 will be the first CMake to support it, though scikit-build-core 0.2+ will backport support to 3.15+.

**Comment by doomroep5** [comment]
> Is there a recommended way to use scikit-build to compile ABI3 wheels? Ideally in a way that is compatible with [cibuildwheel](https://cibuildwheel.readthedocs.io/en/stable/faq/#abi3)?

**Reply by henryiii** [reply]
Scikit-build-core supports this in 0.2+. It does so by backportong FindPython from CMake 3.26.
